nemo/collections/asr/models/ctc_bpe_models.py

```diff
@@ -156,6 +156,86 @@ def _setup_dataloader_from_config(self, config: Optional[Dict]):
             pin_memory=config.get('pin_memory', False),
         )
 
+    def _setup_pseudo_label_dataloader(
```

Comment:
Is there a way we can make the `psuedo_label_dataloader` part of a mixin? This is taking up a decent amount of locs for a niche case.

Reply:
Yes, I believe we can.
Just need to understand how we can understand model is BPE or not.
Do you know how I can do that?

Reply:
If that's simpler, can we support IPL with BPE models only? I'm not sure that non-BPE models are popular anymore (CC @titu1994 @nithinraok WDYT?). Otherwise you can check for `hasattr(self, "tokenizer")` as a somewhat hacky check.

Reply:
I think BPE is a fair assumption. Character level labels are mostly a legacy thing at this point anyhow. I also wouldn't be suprised if IPL just doesn't even work without tokenization at the beginning to 'kick-start' things. 

Actually, separate PR convo, but @maybe we should just start phasing out the character level dataloaders in general? I don't think they're used and they just double any dataloader work. Unless you're doing unsupervised stuff, everything I see nowadays has a some subword tokenziation. And if it's a big issue, then you can just use our current BPE setup to generate a trivial character level vocabulary anyhow. 

Also, potentially less hacky, anything that's BPE would have a `ASRBPEMixin`, could also just check for that inheritance?

Reply:
Moved pseudo label dataloaders to Mixin, but sticked with `hasattr(self, "tokenizer") `, as `ASRBPEMixin` is also inherited in `EncDecHybridRNNTCTCModel`